### PR TITLE
added read me for two tests:office-addin-test-helpers and office-addi…

### PR DESCRIPTION
**Pull request by julflore000**
This addresses https://github.com/OfficeDev/Office-Addin-Scripts/issues/137

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ The [Excel Custom Functions](https://github.com/OfficeDev/Excel-Custom-Functions
 * [office-addin-node-debugger](packages/office-addin-node-debugger/README.md)
 
   This package allows a Node instance to serve as a proxy for debugging a JavaScript runtime hosted by an Office application. 
+  
+* [office-addin-test-helpers](packages/office-addin-test-helpers/README.md)
+
+  This package provides test helpers for automating validation of Office Addins. Use in conjunction with the office-addin-test-server package and Mocha tests.
```
Comment (akrantz) [inline README.md]: This isn't dependent upon the test framework (Mocha). The wording could be revisted for clarity. @mscharlock @davidchesnut would appreciate your feedback.

File: README.md [hunk 2]
```diff
@@ -39,6 +39,15 @@ The [Excel Custom Functions](https://github.com/OfficeDev/Excel-Custom-Functions
...
+
+* [office-addin-test-server](packages/office-addin-test-server/README.md)
+
+  This package provides a framework for testing Office Taskpane Add-ins by allowing Add-ins to send results to a test server.  The results can then be consumed and used
```
Comment (akrantz) [inline README.md]: I think you should remove the hard-coded newline to allow the line to wrap naturally. This wording here seems good to me.

**Comment by akrantz** [comment]
I'm still reviewing this.

File: README.md (hunk 2 above)
Comment (julflore000) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (mscharlock) [reply README.md]:
```suggestion
  This package provides tools that make validating your Office Add-in easier. You can use it with the office-addin-test-server package and the Mocha test framework (or another testing framework of your choice). 
```

File: README.md [hunk 3]
```diff
@@ -39,6 +39,14 @@ The [Excel Custom Functions](https://github.com/OfficeDev/Excel-Custom-Functions
...
 
   This package allows a Node instance to serve as a proxy for debugging a JavaScript runtime hosted by an Office application. 
+  
+* [office-addin-test-helpers](packages/office-addin-test-helpers/README.md)
+
+  This package provides test helpers for automating validation of Office Addins. Use in conjunction with the office-addin-test-server package and Mocha tests.
+
+* [office-addin-test-server](packages/office-addin-test-server/README.md)
+
+  This package provides a framework for testing Office Taskpane Add-ins by allowing Add-ins to send results to a test server. The results can then be consumed and used by tests to validate that the Add-in is working as expected.
```
Comment (davidchesnut) [inline README.md]:
```suggestion
  This package provides a framework for testing Office task pane add-ins by allowing add-ins to send results to a test server. The results can then be consumed and used by tests to validate that the add-in is working as expected.
```

File: README.md [hunk 4]
```diff
@@ -39,6 +39,14 @@ The [Excel Custom Functions](https://github.com/OfficeDev/Excel-Custom-Functions
 * [office-addin-node-debugger](packages/office-addin-node-debugger/README.md)
 
   This package allows a Node instance to serve as a proxy for debugging a JavaScript runtime hosted by an Office application. 
+  
+* [office-addin-test-helpers](packages/office-addin-test-helpers/README.md)
+
+  This package provides test helpers for automating validation of Office Addins. Use in conjunction with the office-addin-test-server package and Mocha tests.
```
Comment (davidchesnut) [inline README.md]:
```suggestion
  This package provides test helpers for automating validation of Office Add-ins. Use in conjunction with the office-addin-test-server package and Mocha tests.
```

File: README.md [hunk 5]
```diff
@@ -39,6 +39,14 @@ The [Excel Custom Functions](https://github.com/OfficeDev/Excel-Custom-Functions
 * [office-addin-node-debugger](packages/office-addin-node-debugger/README.md)
```
Comment (davidchesnut) [inline README.md]: There is no readme so this link is currently broken

**Comment by davidchesnut** [comment]
@julflore000 I noticed the summary text for each package is also copied into the package's readme. Can you please be sure the text is the same wherever the text is copy pasted into a package readme?

**Comment by akrantz** [comment]
@julflore000 @TCourtneyOwen Please work to update this so we can complete the pull request.

File: README.md (hunk 1 above)
Comment (TCourtneyOwen) [reply README.md]: incorporated Michelle's suggested wording

File: README.md (hunk 3 above)
Comment (TCourtneyOwen) [reply README.md]: Incorporated Michelle's suggested wording

File: README.md (hunk 5 above)
Comment (TCourtneyOwen) [reply README.md]: fixed locally

**Pull request merged**